tests/test_e2e_15_maintenance.py

```diff
@@ -120,3 +120,35 @@ def test_010_create_mw_on_switch_should_move_evc(self):
         h11.cmd('ip link del vlan100')
         h3.cmd('ip link del vlan100')
 
+    def test_011_delete_mw_on_switch(self):
+        # 0. Start maintenance window
+        start = datetime.now() + timedelta(seconds=60)
+        end = start + timedelta(seconds=60)
+        payload = {
```

Comment:
Even though a description is not mandatory for a MW, due to an issue (https://github.com/kytos/maintenance/issues/28) the test will not work if you dont provide the description

```suggestion
        payload = {
            "description":  "mw for test 1",
```